chaosdatadog/metrics/probes.py

```diff
@@ -1,21 +1,25 @@
 from datetime import datetime
-from typing import Literal
 
 from chaoslib.exceptions import ActivityFailed
 from chaoslib.types import Configuration, Secrets
 from datadog_api_client.exceptions import ApiTypeError, NotFoundException
 from datadog_api_client.v1.api.metrics_api import MetricsApi
 from dateutil.relativedelta import relativedelta
+from logzero import logger
 
 from chaosdatadog import get_client
-from chaosdatadog.metrics.utils import extract_metric_name
+from chaosdatadog.metrics.utils import (
+    check_comparison_values,
+    extract_metric_name,
+    get_comparison_operator,
+)
 
 __all__ = ["get_metrics_state"]
 
 
 def get_metrics_state(
     query: str,
-    comparison: Literal[">", "<", ">=", "<=", "=="],
+    comparison: str,
```

Comment:
good call, I would also prefer the `Literal` type but it appeared in Python 3.8.

Mind you Python 3.7 is EOL soon so we'll end up the year with supporting 3.8 only. For now, we must keep `str`.